htdocs/reception/stats/month.php

```diff
@@ -49,9 +49,8 @@
 $mesg = '';
 
 print load_fiche_titre($langs->trans("StatisticsOfReceptions").' '.GETPOST("year", 'int'), $mesg);
-
-$stats = new ReceptionStats($db);
-$data = $stats->getNbReceptionByMonth(GETPOST("year", 'int'));
+$stats = new ReceptionStats($db, $socid, '', ($userid > 0 ? $userid : 0));
```

Comment:
You introduce the parameter $socid.
Yes, the method need it. But it is not defined previously (at least not always).
You must add
$socid = GETPOST('socid', 'int') at line 32 because the id of the company come from a parameter into the form.

Same for $userid.  Where is this var set ? It miss
$userid = GETPOST('userid', 'int')  
that also come as parameter on page:

![image](https://user-images.githubusercontent.com/883887/206217842-f9fe7b7f-72f9-4f28-ac66-c0a63165d1a9.png)